src/main/scala/com.snowplowanalytics.iglu/ctl/Server.scala

```diff
@@ -141,18 +140,24 @@ object Server {
   }
 
   /**
-    * Build HTTP POST-request with JSON Schema and authenticated with temporary
+    * Build HTTP PUT-request with JSON Schema and authenticated with temporary
     * write key
     *
     * @param schema valid self-describing JSON Schema
     * @param writeKey temporary apikey allowed to write any Schema
-    * @return HTTP POST-request ready to be sent
+    * @return HTTP PUT-request ready to be sent
```

Comment:
Why did we need to change it from `POST`?

Reply:
Or was it already `PUT` but just a typo in the scaladoc?

Reply:
Yep, indeed it was a `PUT` [previously](https://github.com/snowplow-incubator/igluctl/pull/118/files#diff-2349fc37156c7a5c007e41c57a52a79a244612c3265d1c7531e9cab064e5ed74L155) and the typo was in the scaladoc